### PR TITLE
[IMP] marketing_automation: whatsappp integration

### PR DESCRIPTION
**Pull request by datonefaridze**
Whatsapp was added as an activity type in marketing automation. Whatsapp addition caused the addition of certain trigger types, as well as integrating whatsapp with link tracker.

task-3595515

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/link_tracker/models/link_tracker.py [hunk 1]
```diff
@@ -316,7 +316,7 @@ class LinkTrackerClick(models.Model):
 
     campaign_id = fields.Many2one(
         'utm.campaign', 'UTM Campaign', index='btree_not_null',
-        related="link_id.campaign_id", store=True, ondelete="set null")
+        default="link_id.campaign_id", store=True, ondelete="set null")
```
Comment (tde-banana-odoo) [inline addons/link_tracker/models/link_tracker.py:319]: Not sure why this change, but this seems louche ^^ . Not sure it even works as intended.

Comment (datonefaridze) [reply addons/link_tracker/models/link_tracker.py:319]: @tde-banana-odoo good, point, i wanted to re-check this for a long time. i have changed code lot of times and maybe this is leftover code (will take a look soon)

Comment (datonefaridze) [reply addons/link_tracker/models/link_tracker.py:319]: @tde-banana-odoo it seems this one was leftover code, therefore I removed branch. (I tested code thoroughly and also logically, didn't find any issue)
